src/main/kotlin/lsifjava/JDK8CompatFileManager.kt

```diff
@@ -0,0 +1,164 @@
+@file:Suppress("JAVA_MODULE_DOES_NOT_EXPORT_PACKAGE")
+package lsifjava
+
+import javax.tools.ForwardingJavaFileManager
+import javax.tools.StandardJavaFileManager
+import com.sun.tools.javac.util.Context
+import java.io.BufferedReader
+import java.io.File
+import java.io.IOException
+import java.io.InputStreamReader
+import java.nio.charset.Charset
+import java.nio.file.*
+import java.util.concurrent.TimeUnit
+import java.util.stream.Collectors
+import javax.tools.JavaFileManager
+import javax.tools.StandardLocation
+
+/**
+ * FileManager that falls back to JavacPathFileManager for Java 8.
+ * Java 8 StandardJavaFileManager doesn't have the <code>setLocationFromPaths</code>
+ * method, instead it only has <code>setLocation</code> which requires an 
+ * <code>Iterable<? extends File></code>, which would cause an UnsupportedException
+ * when trying to turn the ZipFile from the in-memory FileSystem into a File.
+ * Because JavacPathFileManager doesnt exist beyond Java 8 (9?) and we build with 14+,
+ * the symbol resolver would fail for that symbol, hence we create an instance via
+ * reflection if the Java version is 8. God I hate this
```

Comment:
```suggestion
 * reflection if the Java version is 8. God I hate this.
```

Let's be professional, Noah.

Reply:
:man_facepalming: dont know how I missed that. How did I get this job again